src/Tes.Runner/Transfer/BlobBlockApiHttpUtils.cs

```diff
@@ -5,6 +5,7 @@
 using System.Net.Http.Headers;
 using System.Net.Sockets;
 using System.Text;
+using System.Web;
```

Comment:
I don't think we want this dependency in ASP.NET Core... do we really need System.Web ?

Reply:
URL encoding for the tags.